engine/src/main/resources/application.conf

```diff
@@ -32,6 +32,9 @@ spray.can {
 system {
   // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
   abort-jobs-on-terminate = false
+
+  // Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
+  max-retries = 3
```

Comment:
Think @kcibul was mentioning somewhere around 10-20. Do you think 10 should be fine?